server/protocols/jmap/src/test/java/org/apache/james/jmap/methods/GetMailboxesMethodTest.java

```diff
@@ -225,20 +225,20 @@ public void getMailboxesShouldReturnInboxWithSortOrder10OnDifferenceCase() throw
                 .extracting(GetMailboxesResponse.class::cast)
                 .flatExtracting(GetMailboxesResponse::getList)
                 .extracting(Mailbox::getSortOrder)
-                .containsOnly(SortOrder.of(10));
+                .containsOnly(SortOrder.of(1000));
```

Comment:
as InBoX should be the same than Inbox, this tests should not be changed anymore, no?